worker/include/RTC/PlainTransport.hpp

```diff
@@ -76,8 +76,12 @@ namespace RTC
 		ListenInfo rtcpListenInfo;
 		bool rtcpMux{ true };
 		bool comedia{ false };
-		struct sockaddr_storage remoteAddrStorage;
-		struct sockaddr_storage rtcpRemoteAddrStorage;
+		struct sockaddr_storage remoteAddrStorage
+		{
+		}; // NOLINT
```

Comment:
This NOLINT is intended for lint or for tidy task?

Reply:
These where indeed no needed. Removed.